doc/users/install.rst

```diff
@@ -107,7 +107,7 @@ recommendations.
 Must Have
 ~~~~~~~~~
 
-Python_ 2.6 - 2.7
+Python_ 2.7
 
 Nibabel_ 1.0 - 1.4
```

Comment:
yes - but let's do this in a separate PR

On Wed, Mar 18, 2015 at 3:44 PM, Satrajit Ghosh <notifications@github.com>
wrote:

> In doc/users/install.rst
> <https://github.com/nipy/nipype/pull/1069#discussion_r26715368>:
>
> > @@ -107,7 +107,7 @@ recommendations.
> >  Must Have
> >  ~~~~~~~~~
> >
> > -Python_ 2.6 - 2.7
> > +Python_ 2.7
> >
> >  Nibabel_ 1.0 - 1.4
>
> should we update the ranges for the rest of the libraries?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/nipype/pull/1069/files#r26715368>.
>